common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -255,6 +255,98 @@ color_lists = {
 			125 = "black"
 			250 = "black_empire_dark_red"
 		}
+		
+		###### Warcraft Culture Specific CoAs ######
+		
+		### Humans ###
+		
+		special_selection = { 
+			trigger = { azerothian_coa_trigger = yes }
+			15  = "green_tirassian"
+			15  = "cyan_silverhand"
+			20  = "purple"
+			20  = "red"
+			150 = "blue"
+		}
+		special_selection = {
+			trigger = { lordaeronian_coa_trigger = yes }
+			10  = "purple"
+			10  = "green_tirassian"
+			15  = "cyan_silverhand"	
+			25  = "red"
+			65  = "grey_dark"
+			150 = "blue"
+		}	
+		special_selection = {
+			trigger = { arathorian_coa_trigger = yes }
+			15  = "purple"
+			25  = "black"
+			45  = "brown_strom"	
+			200 = "red"	
+		}
+		special_selection = {
+			trigger = { alteraci_coa_trigger = yes }
+			15  = "purple"			
+			50  = "black"
+			50  = "orange_alterac"
+			50  = "brown_alterac"
+		}
+		special_selection = {
+			trigger = { gilnean_coa_trigger = yes }
+			10  = "purple_tidesage"
+			10  = "aqua_stormsong"
+			10  = "brown_waycrest"	
+			15  = "green_tirassian"
```

Comment:
I can understand why it's so since Kul Tirans come from Gilneas but I would rather focus on making all the cultures destinct.

Reply:
How would you like me to proceed here? Reduce weights further (effectively keeping them at this level if I increase the main culture colours to 250) or remove them entirely?

Reply:
I would remove it. As I said, I would focus on uniquness.